pyaedt/application/Design.py

```diff
@@ -588,12 +588,16 @@ def __init__(
         self._oimport_export = self._desktop.GetTool("ImportExport")
         self._odefinition_manager = self._oproject.GetDefinitionManager()
         self._omaterial_manager = self.odefinition_manager.GetManager("Material")
-        self.odesktop = self._desktop
         self._variable_manager = VariableManager(self)
         self.solution_type = self._solution_type
         self.project_datasets = self._get_project_datasets()
         self.design_datasets = self._get_design_datasets()
 
+    @property
+    def odesktop(self):
+        """Desktop instance containing all projects and designs."""
+        return self._desktop
```

Comment:
I'm of the opinion that "odesktop" isn't a python friendly attribute. Recommending renaming to `desktop`. The leading "o" won't make sense for new users of PyAEDT.

We should also provide a trivial example and return value for this.

Reply:
@akaszynski  I named it "odesktop" because it returns a COM object and that have been the naming convention applied to the entire project. Should I get rid of the convention for this property or get the name more pythonic?
I will add example and return value.

Reply:
Good question. Whatever we do should be consistent, and if we have a standard to use "o" for COM objects, let's use it.

Reply:
Ok, so I will use `odesktop` then.

Reply:
Still needs an example I think, unless we don't intend for the user to access this attribute.